Engine/ac/characterextras.cpp

```diff
@@ -34,6 +34,22 @@ void CharacterExtras::ReadFromFile(Stream *in)
     process_idle_this_time = in->ReadInt8();
     slow_move_counter = in->ReadInt8();
     animwait = in->ReadInt16();
+    if (cmp_ver >= 2)
```

Comment:
Please give new save format a much higher number in case there will be change in 3.* branch